src/DetailsView/components/assessment-test-view.tsx

```diff
@@ -27,7 +28,9 @@ import { TabStoreData } from '../../common/types/store-data/tab-store-data';
 import { VisualizationStoreData } from '../../common/types/store-data/visualization-store-data';
 import { AssessmentInstanceTableHandler } from '../handlers/assessment-instance-table-handler';
 
-export type AssessmentTestViewDeps = ScanIncompleteWarningDeps &
+export type AssessmentTestViewDeps = {
```

Comment:
`getProvider` exists in`RequirementViewDeps` so we shouldn't need this addition.

Reply:
For deps, we usually specifically call out the deps in the component that we're defining even if a child component might also be using the same deps. This prevents cascading type errors if you remove one usage of a dependency somewhere within the component tree.